test/suite/wttest.py

```diff
@@ -717,8 +717,8 @@ def timestamp_str(self, t):
 
     # Some tests do table drops as a means to perform some test repeatedly in a loop.
     # These tests require that a name be completely removed before the next iteration
-    # can begin.  However, tiered storage does not always provide a way to remove or
-    # rename objects that have been stored to the cloud, as doing that is not the normal
+    # can begin.  However, tiered storage does not always provide a way to remove
```

Comment:
I think the word "objects" was accidentally also consumed here.

Reply:
good spotting, [4651212](https://github.com/wiredtiger/wiredtiger/pull/10282/commits/4651212791deac3577df5658fab5aa3c0f6f45c4)